addons/crm/crm_lead_menu.xml

```diff
@@ -76,10 +76,46 @@
 
     <menuitem id="base.menu_sales" name="Sales" parent="base.menu_base_partner" sequence="5"/>
     <menuitem name="Leads" id="menu_crm_leads" parent="base.menu_sales" sequence="1"
-            action="crm_lead_all_leads"/>
-    <menuitem name="Opportunities" id="menu_crm_opportunities" parent="base.menu_sales" sequence="2"
+            action="crm_lead_all_leads" groups="crm.group_use_lead"/>
+    <menuitem name="Opportunities Pipeline" id="menu_crm_opportunities" parent="base.menu_sales" sequence="2"
             action="crm_lead_opportunities"
             groups="base.group_sale_salesman,base.group_sale_manager"/>
 
+    <record id="crm_lead_next_action" model="ir.actions.act_window">
+        <field name="name">Next Actions</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">crm.lead</field>
+        <field name="view_mode">tree,form</field>
+        <field name="search_view_id" ref="crm.crm_lead_next_action_search_view"/>
+        <field name="domain">[('type','=','opportunity'), ('date_action','!=',False)]</field>
+        <field name="context">{
+                'stage_type': 'opportunity',
+                'default_type': 'opportunity',
+                'default_user_id': uid
+            }
+        </field>
+        <field name="help" type="html">
+            <p>
+               Here, you can see your next actions.
```

Comment:
Change into : Here is the list of your next actions. Those are linked to your opportunities. To set a next action, go on an opportunity and add one. It will then appear in this list.